sync/p2p_messages.md

```diff
@@ -139,6 +150,36 @@ Message is RLP encoded, fields:
 A signed transaction is serialized as a tagged and versioned
 [signed transaction](../serializations.md#signed-transaction).
 
+## MSG_BLOCK_TXS
+*(Tag = 13)
+
+Message is RLP encoded, fields:
+  - `Hash :: byte_array - The block we fetch TXs from`
+  - `Txs :: [byte_array] - List of serialized signed TXs`
+
+A signed transaction is serialized as a tagged and versioned
+[signed transaction](../serializations.md#signed-transaction).
+
+## MSG_KEY_BLOCK
+*(Tag = 10)
+
+Message is RLP encoded, fields:
+  - `KeyBlock :: byte_array - Serialized key block`
+
+The key block is [serialized](../serializations.md#key-block-header).
+
+## MSG_MICRO_BLOCK
+*(Tag = 11)
+
+Message is RLP encoded, fields:
+  - `MicroBlock :: byte_array - Serialized micro block`
+  - `Light :: bool - flag if micro block is light or normal`
+
+A normal micro block is [serialized](../serializations.md#micro-block-header).
```

Comment:
The source for this is
https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_peer_connection.erl#L953-L954
that ends up in
https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/src/aec_blocks.erl#L338-L343

so correct link seems [`/serializations.md#micro-block`](https://github.com/aeternity/protocol/blob/836975660a6ff94888930f9d82ebb2db1a54ea95/serializations.md#micro-block) ? cc @gorillainduction